### DISCUSSION TITLE
downsampling

### DISCUSSION THREAD

**Discussion by SGKhir** [ImmuneDynamics/Spectre#146] [Q&A]
Hi Thomas,

Brilliant tool. In the simple discovery workflow https://immunedynamics.io/spectre/simple-discovery/ I dont understand on how did you end up choosing 2000 and 20000 as cell counts for downsampling? Could you please clarify? Thanks.

**Comment by ghar1821** [comment]
Hi @SSKhir ,

I think @tomashhurst just picked a number that is lower than the number of cells in each group. The groups are downsampled so we don't overcrowd the umap/tsne plots. It is also so the umap/tsne calculation doesn't take overly long - they can take a long time if we have too many cells. I guess you can pick any number as long as it's lower than the number of cells in each group.

**Comment by tomashhurst** [comment]
@SSKhir that's a great question! It's actually a reflection of the relative cell numbers in each group. The cells present in the WNV samples are ~10x the number of cells in the mock samples (i.e. the total number of cells in the tissue sample itself), so keeping this ration in mind, for downsampling, if mock contains 2000 cells, then WNV should contain 20,000. 

The paper discusses it, but this is really something we should write up a more detailed tutorial about. In most cases, if people are analysing PBMCs or such, then folks just pick the same number of cells for each group.

**Reply by csecarroll** [reply]
I swear there was an excellent tutorial on this on the website but the link appears broken and I can't find it anywhere. Is it possible to get a link to this?
